tests/middleware/test_middleware.py

```diff
@@ -0,0 +1,95 @@
+"""Import the Middleware class from the corresponding module"""
+import pytest
+from app.middleware.request_validations import Middleware
+from ..resources import DATA
```

Comment:
```suggestion
from app.tests.resources import DATA
```

Es mejor manejar el path global (no estoy seguro si este es el path, lo puse como creí)